lib/data_services_api/version.rb

```diff
@@ -2,6 +2,6 @@
 
 # :nodoc:
 module DataServicesApi
-  VERSION = '0.4.5'
+  VERSION = '1.1.0'
```

Comment:
Odd jump

Reply:
Yes, a bit unfortunate. This is a long-running branch that was set up to help the new LR infra work, and had multiple updates to the API-wrapper gem. We bumped to 1.0.0 on the initial update, and then subsequent new features added. Merging back to master probably should have happened a while ago.